config/chainbridge-runtime-config.cere.dev.js

```diff
@@ -32,6 +32,13 @@ window.__RUNTIME_CONFIG__ = {
             isDoubleApproval: false,
           },
         ],
+        transferFallback: [
+          {
+            chainId: 1,
+            delayMs: 4 * 60 * 1000,
+            pollingIntervalMs: 2000,
```

Comment:
Polling interval should depend on block time in the destination chain. 
I would say it should be smth like `Math.Min(Math.Max(15s, 3*destination_chain_block_time), 30s)`

Reply:
What is destination chain block time?

Reply:
resolved